migration/template/apply/00-template.sh

```diff
@@ -48,9 +48,23 @@ run() {
   execute)
     # Note: 00-template.sh will be skipped by the upgrade command
     log_info "no operation: this is a template"
+
+    if [[ "${CK8S_CLUSTER:-}" =~ ^(sc|both)$ ]]; then
+      log_info "operation on service cluster"
+    fi
+    if [[ "${CK8S_CLUSTER:-}" =~ ^(wc|both)$ ]]; then
```

Comment:
We need to have it so that the loading of the lib will check and error out if the that env isn't set, so we should never need to guard for it here.

And we need a note for the manual steps that you need to set it to target the proper cluster.

Reply:
I think the check that is already in place should be sufficient for verifying if `CK8S_CLUSTER` is set or not. I updated the manual migration steps  and removed guards from the snippets. Please have a look.